pytensor/tensor/rewriting/linalg.py

```diff
@@ -310,3 +311,27 @@ def local_log_prod_sqr(fgraph, node):
 
         # TODO: have a reduction like prod and sum that simply
         # returns the sign of the prod multiplication.
+
+
+cholesky_no_inplace = Cholesky(overwrite_a=False)
+cholesky_inplace = Cholesky(overwrite_a=True)
+
+
+@node_rewriter([cholesky_no_inplace], inplace=True)
+def local_inplace_cholesky(fgraph, node):
+    new_out = [cholesky_inplace(*node.inputs)]
+    copy_stack_trace(node.outputs, new_out)
+    return new_out
+
+
+# After destroyhandler(49.5) but before we try to make elemwise things
+# inplace (75)
+linalg_opt_inplace = in2out(local_inplace_cholesky, name="linalg_opt_inplace")
+optdb.register(
+    "InplaceLinalgOpt",
+    linalg_opt_inplace,
+    "fast_run",
+    "inplace",
```

Comment:
Using the term inplace in many places is perhaps a good argument to also call it inplace internally?

Reply:
My preferences on the matter are `overwrite_a=True, overwrite_b=True > inplace_a=True, inplace_b=True > inplace = (True, True) > destroy_a = True, destroy_b = True > destroy = (True, True)`

Reply:
Just because we refer to it as inplace everywhere (except destroy_map) and that's how you exclude inplace rewrites, but not a strong preference. 

Just no point in the internal prop being called something other than the init argument. If you call it overwrite_a, make self.overwrite_a also instead of self.inplace so that we can easily automate the rewrites.